view/src/storage.rs

```diff
@@ -816,6 +955,7 @@ impl Storage {
             )
             .fetch_optional(&mut dbtx)
             .await?;
+            // TODO: mark spent swaps as spent
```

Comment:
Followup: mark claimed `Swap`s in the view server

Currently, although we have a `height_claimed` field in the `SwapRecord`, we never set it, and never mark the swap as claimed. We should use similar logic as for notes to update `SwapRecord`s with the claim height.